Tribler/Main/vwxGUI/CreditMiningPanel.py

```diff
@@ -0,0 +1,533 @@
+"""
+This module contains credit mining panel and list in wx
+
+Written by Ardhi Putra Pratama H
+"""
+
+import logging
+import wx
+from binascii import hexlify
```

Comment:
What IDE are you using? With PyCharm, I can just go to `code` -> `Optimise imports`.